src/components/reviews/reviews.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ReviewsColumn from "../reviews-column/reviews-column.jsx";
+import {movieShape} from "../shapes";
+
+const splitReviewsIntoColumns = (reviews) => {
+  const firstColumnLength = Math.ceil(reviews.length / 2);
+  const firstColumnReviews = reviews.slice(0, firstColumnLength);
+  const secondColumnReviews = reviews.slice(firstColumnLength);
+
+  return [firstColumnReviews, secondColumnReviews];
+};
+
+const Reviews = (props) => {
+  const {movie} = props;
+  const {reviews} = movie;
```

Comment:
можно делать множественную деструктуризацию в одну строку, так как `movie` ты нигде в компоненте не используешь: 
```suggestion
  const {movie: { reviews }} = props;
```